src/resolvers/Inventory.ts

```diff
@@ -0,0 +1,15 @@
+import Inventories from "../services/Inventories";
+
+const resolvers = {
+  Inventory: {},
```

Comment:
I don't know if you've implemented the fix I mentioned but just make sure you fill out the product field so that when you query an inventory you're able to also see the product field. It's kind of like what you did in Product.ts.
`Product: {
    inventory: ({ id }) => Inventories.find({ id }),
  },`